### ISSUE TITLE
Add usecases

### ISSUE THREAD

**Issue by ninsbl** [OSGeo/grass-website#28]
Where? Learn?
https://grass.osgeo.org/documentation/applications/

**Comment by lucadelu** [comment]
yes this should go in learn, but use cases should be something more then applications, something like [QGIS use cases](https://www.qgis.org/en/site/about/case_studies/index.html)

**Comment by veroandreo** [comment]
@lindakarlovska suggested this: https://airflow.apache.org/use-cases/ during the community meeting in Prague. See her message [here](https://matrix.to/#/!BOjyJgENYOLyXbRKxO:gitter.im/$6EX0NIht50UIbjuWAKygIlyhaQCyFCpFItwP9b1xBEQ?via=gitter.im&via=matrix.org&via=osgeo.org).
